### DISCUSSION TITLE
Would it be possible to use Wi-Fi for faster and continuous streaming of IO on Pico Pi W boards?

### DISCUSSION THREAD

**Discussion by JeffreyO** [gusmanb/logicanalyzer#53] [Ideas]
At least, assuming that this is limited by the bandwidth of USB2 like Salae16 clones. Even if bandwidth isn't possible to increase, it could open up a more flexible way to put the Pi Pico W anywhere without needing to worry about a USB cable.

**Comment by gusmanb** [comment]
WiFi IO is slower, at least on my tests USB was faster. And about using it without the USB cable... you already can do it, you can use the W wirelessly, in fact I use it like that, I have a 13650 shield and I power the W from there so I don't need any wire at all.

This explains how to build each version of the firmware, including the wireless version: https://github.com/gusmanb/logicanalyzer/wiki/03---LogicAnalyzer-Firmware

This explains how to connect to the W wirelessly: https://github.com/gusmanb/logicanalyzer/wiki/06---The-LogicAnalyzer-program#the-network-device

And here you can find the pre-built firmware: https://github.com/gusmanb/logicanalyzer/releases/tag/v4.0.0.0 It's the "W-WiFi" version.

Cheers.
